addon/components/materialize-collection-item.js

```diff
@@ -0,0 +1,14 @@
+import Ember from 'ember';
+import layout from '../templates/components/materialize-collection-item';
+
+export default Ember.Component.extend({
+  layout: layout,
+  tagName: 'li',
+  isActive: false,
+  isHeader: false,
+  isDismissable: false,
+  image: '',
+  imageAlt: '',
+  title: '',
```

Comment:
For consistency with the rest of the addon, please use null instead of empty strings for "empty" default values whenever possible.